docs/research.md

```diff
@@ -0,0 +1,33 @@
+# Research & Discovery
+
+## Existing similar tools and benefits to our project
+
+### 0xFableOrg/roll-op
+- Rollup customization with config file
+
+
+### upnodedev/opstack-compose
+- Dev ops solution (docker-compose + aws)
+- Upnode focuses on blockchain infrastructure (more value)
+- Has a block explorer
+
+## Decisions and drivers
+
+### Developed on Rust
+- Fast, lightweight and compiled
+- Reduces computing costs
+- Increasing popularity
```

Comment:
Maybe specify why not `go` also, as optimism is in go, we can say like the `op-deployer` can potentially be a go alternative and in this case we're not making a pr to the optimism project itself but living as an individual tool.